test/autorest/generated/morecustombaseurigroup/client.go

```diff
@@ -69,22 +67,19 @@ func NewClient(endpoint string, options *ClientOptions) (*Client, error) {
 		azcore.NewUniqueRequestIDPolicy(),
 		azcore.NewRetryPolicy(&options.Retry),
 		azcore.NewRequestLogPolicy(options.LogOptions))
-	return NewClientWithPipeline(endpoint, p)
+	return NewClientWithPipeline(p)
 }
 
 // NewClientWithPipeline creates an instance of the Client type with the specified endpoint and pipeline.
-func NewClientWithPipeline(endpoint string, p azcore.Pipeline) (*Client, error) {
-	u, err := url.Parse(endpoint)
-	if err != nil {
-		return nil, err
-	}
-	if u.Scheme == "" {
-		return nil, fmt.Errorf("no scheme detected in endpoint %s", endpoint)
-	}
-	return &Client{u: u, p: p}, nil
+func NewClientWithPipeline(p azcore.Pipeline) (*Client, error) {
+	return &Client{p: p}, nil
 }
 
 // PathsOperations returns the PathsOperations associated with this client.
-func (client *Client) PathsOperations(dnsSuffix string, subscriptionID string) PathsOperations {
-	return &pathsOperations{Client: client, dnsSuffix: dnsSuffix, subscriptionID: subscriptionID}
+// For dnsSuffix pass nil to use the default value of "host"
+func (client *Client) PathsOperations(dnsSuffix *string, subscriptionID string) PathsOperations {
```

Comment:
Why is `dnsSuffix` on the operation group here?  It's a client param that's common to all operation groups.

Reply:
You're right I was missing that check it's added now. Shared client params are now on the client in the case of parameterized host